images/flink/README.MD

```diff
@@ -0,0 +1,138 @@
+# Save savepoints to GCS
+
+Flink Operator supports taking savepoints and storeing it into Google Cloud Storage when deploying a job with a custom flink image with GCS connector installed. 
+
+## Create custom docker image with GCS connector
+
+Note that there are two ways to provide Flink the job jar. You can directly embed it into the image, e.g. COPY from a local file path in Dockerfile. You can also upload your job jar onto GCS and then reference it in your job YAML file with `jarFile: gs://my-bucket/path-to-jar`. This example takes the first approach and supports downloading the job jar from GCS and putting it under `${FLINK_HOME}/job/` of the image, the job jar URI is set in ${FLINK_JOB_JAR_URI}. You can customize the entrypoint.sh or Dockerfile to upload your job jar in your own way.
+
+After the ${FLINK_JOB_JAR_URI} is set correctly if you also upload job jar in the same way as in example, you can simply run the command below to build the image and push it to GCR.
+
+```bash
+make build push IMAGE_TAG=gcr.io/google.com/<my-project>
+```
+
+The default flink version or gcs connector version can be modified in properties file as you need.
```

Comment:
s/flink/Flink, s/gcs/GCS and other occurrences.

Reply:
Done.